src/panel/poi_panel.js

```diff
@@ -68,6 +69,7 @@ PoiPanel.prototype.close = async function() {
 }
 
 PoiPanel.prototype.restorePoi = async function (id) {
+  Telemetry.add(Telemetry.POI_RESTORE)
```

Comment:
`poi_restore` event is undefined.  
This causes a request without body to backend, that returns 400. 
I think `Telemetry.add` should validate the event, and log invalid events instead of triggering an invalid request.

Reply:
good idea